Completion/Unix/Command/_find

```diff
@@ -46,7 +48,7 @@ case $variant in
       '*-anewer:file to compare (access time):_files'
       '*-cnewer:file to compare (inode change time):_files'
       '*-empty'
-      '*-execdir:program: _command_names -e:*\;::program arguments: _normal'
+      '*-execdir:program: _command_names -e:*(\;|+)::program arguments: _normal'
```

Comment:
As below.

Reply:
Again, this seems to be working perfectly for me, really confused as to what the issue could be.